pyproject.toml

```diff
@@ -105,6 +106,30 @@ exclude_lines = [
   "@overload",
 ]
 
+[tool.mypy]
+  python_version           = "3.11"
+  files                    = ["quantity"]
+  strict                   = true
+
+  disallow_incomplete_defs = true
+  disallow_untyped_defs    = false
+  enable_error_code        = ["ignore-without-code", "redundant-expr", "truthy-bool"]
+  warn_return_any          = true
+  warn_unreachable         = true
+  warn_unused_configs      = true
+
+  [[tool.mypy.overrides]]
+    module = ["quantity._dev.*", "quantity.tests.*"]
```

Comment:
Just a bit of a flyby - cool to see the `mypy` setup! But this is out of date - test has moved now (and another reason why that is good!)